Tests/ClientTests/TelemetryWrapperTests.swift

```diff
@@ -468,11 +468,11 @@ class TelemetryWrapperTests: XCTestCase {
 
 // MARK: - Helper functions to test telemetry
 extension XCTestCase {
-    func testEventMetricRecordingSuccess<Keys: EventExtraKey, Extras: EventExtras>(
```

Comment:
These changes are to allow for a bump in a-s and glean.

Reply:
If you rebase with `main` we already use 97.1.0 A-S and Glean 52.3.0 so you'll get those for free.

Reply:
@lmarceau Since I'm pushing to the epic branch as a part of the iOS13 cutoff I wasn't sure how much of main I should be taking.

Reply:
Oh right! I had forgotten you were pushing to the `epic branch`. Well in that case, I would recommend keeping the `epic branch` up to date with main, so that way we ensure there's no merge conflicts when the time comes to bring the changes to main.

Reply:
@lmarceau I don't have permissions to push to the epic branch.

Reply:
Ok I just updated it! I'll try to keep an eye for it so we keep it up to date as much as possible. Thank you!

Reply:
Thanks!!